app/src/features/FhirResourceTree/resourceTreeSlice.ts

```diff
@@ -27,6 +27,7 @@ export type ElementNode = {
 
 type ResourceTreeSliceState = {
   root?: ElementNode;
+  selectedNodeId?: string;
```

Comment:
I think we must use the router for this, not the store, to record the current selected node. When a node is selected, the route would be `/sources/{sourceId}/mappings/{resourceId}/attributes/{attributeId}`. The idea is that we should be able to share a link to the attribute we're working on.